media-gfx/iscan/iscan-2.30.4.2.ebuild

```diff
@@ -126,7 +126,7 @@ src_install() {
 		done
 		if [[ "/plug-ins" != "${gimpplugindir}" ]]; then
 			dodir ${gimpplugindir}
-			dosym "${ED%/}"/usr/bin/iscan "${gimpplugindir}"/iscan
+			dosym $(realpath -s --relative-to="${gimpplugindir}" "${EPREFIX}/usr/bin/iscan") "${gimpplugindir}"/iscan
```

Comment:
@Arfrever what do you think about this? Suitable? 
I'd expect a `|| die` in here, since if it fails to execute, non-relevant info is returned (if any at all).

Reply:
This line was copied from what Arfrever suggested [here](https://bugs.gentoo.org/699498).

Personally I think that ebuilds could just hardcode value `../../../../bin/iscan` calculated using output of `gimptool-2.0 --gimpplugindir` (which is `/usr/lib64/gimp/2.0`), since it most likely will never change. (In GIMP 3, path will be different, but I expect that GIMP 3 will not have `gimptool-2.0` tool anyway.)

So you can choose between:
```
dosym $(realpath -s --relative-to="${gimpplugindir}" "${EPREFIX}/usr/bin/iscan" || die) "${gimpplugindir}"/iscan
```
```
dosym ../../../../bin/iscan /usr/$(get_libdir)/gimp/2.0/plug-ins/iscan
```